src/css/controls/imports/controlbar.less

```diff
@@ -181,6 +180,9 @@
         .jw-svg-icon-live {
             display: none;
         }
+
+        // padding between elapsed time in DVR mode
+        margin-right: 1em;
```

Comment:
We should use a pixel value for the margin. Using `em` would result in the spacing changing if the font size were ever to change, which is not what we want.